reporter/src/main/kotlin/ReportTableModelMapper.kt

```diff
@@ -45,46 +45,6 @@ import org.ossreviewtoolkit.reporter.ReportTableModel.SummaryTable
  * A mapper which converts an [OrtResult] to a [ReportTableModel].
  */
 object ReportTableModelMapper {
-    private fun Issue.toResolvableIssue(
```

Comment:
The `object` now contains only the single `map` function. Instead of using an object to put that single function into a namespace, I wonder whether it should also become a top-level function, and maybe name it back (or not rename it in the first place) to the original `mapToReportTableModel` name, and maybe also make it an extension function on `OrtResult`.

Reply:
I prefer keeping it an object.

Reply:
> It's also consistent with other mappers IIRC.

I'd say we should rather fix then the others, too. But that can happen outside of this PR.

Reply:
Sure out scope. Let's just decide on such style thing together in the Kotlin core devs group in the mondays meeting.